lib/link-constraints.ts

```diff
@@ -1676,6 +1676,46 @@ export const constraints: LinkConstraint[] = [
 			inverse: 'link-constraint-loop-has-product-repository',
 		},
 	},
+	{
+		slug: 'link-contraint-reaction-is-attached-to-message',
+		name: 'is attached to',
+		data: {
+			title: 'Reaction',
+			from: 'reaction',
+			to: 'message',
+			inverse: 'link-constraint-message-has-attached-element-reaction',
+		},
+	},
+	{
+		slug: 'link-constraint-message-has-attached-element-reaction',
+		name: 'has attached element',
+		data: {
+			title: 'Reaction contract',
```

Comment:
```suggestion
			title: 'Reaction',
```

Nit pick, but there's no reason this one should have a different title than others